File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int curr = -101;
        int l = 0;
        
        for(int i = 0; i<nums.length; i++){
            if(curr != nums[i]){
                curr = nums[i];
                nums[l] = curr;
                l += 1;
            }
        }
        return l;
        
        
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    
    int removeDuplicates(vector<int>& nums) {
        int idx = 0;
        for (int i = 1; i < nums.size(); i++){
            if(nums[i] > nums[i-1]){
                idx++;
                nums[idx] = nums[i];
            }
        }
        
        return idx+1;
        
    }
};